appraisal/jruby-9.2.rb

```diff
@@ -202,6 +202,8 @@
 build_coverage_matrix('stripe', 7..12, min: '5.15.0')
 build_coverage_matrix('opensearch', 2..3, gem: 'opensearch-ruby')
 build_coverage_matrix('elasticsearch', 7..8)
+build_coverage_matrix('kicks', [], min: '3.0.0')
+build_coverage_matrix('sneakers', [], min: '2.12.0', latest: false) # Sneakers is not receiving updates anymore and 2.12.0 is the last version
```

Comment:
I actually think we don't need to invoke `build_coverage_matrix` with `latest: false`, since this method call only generate a single group. 
 
Prefer the following for explicitness
 
```ruby
appraisal 'sneakers-min' do 
  gem 'sneakers', '= 2.12.0'
end

```